pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -87,20 +87,25 @@ contract ManagedPool is BaseWeightedPool, ReentrancyGuard {
 
     // Store scaling factor and start/end weights for each token
     // Mapping should be more efficient than trying to compress it further
-    // [ 155 bits|   5 bits |  32 bits   |   64 bits    |
-    // [ unused  | decimals | end weight | start weight |
-    // |MSB                                          LSB|
+    // [ 123 bits |  5 bits  |  64 bits   |   64 bits    |
+    // [ unused   | decimals | end weight | start weight |
+    // |MSB                                           LSB|
```

Comment:
I'm not sure we really need to, but I thought it would make it more accurate/consistent (e.g., when we copy end weights to start weights), now that we could have a wider range of values